financial-connections/src/main/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerViewModel.kt

```diff
@@ -104,8 +104,10 @@ internal class InstitutionPickerViewModel @Inject constructor(
                         resultCount = result.data.count()
                     )
                 )
+                result
+            } else {
+                InstitutionResponse(emptyList())
```

Comment:
in this case will it show empty search result or list of featured institutions when the query is empty? Maybe it's good to show featured institution list if not too hard to implement.

Reply:
featured institutions : )

Just that we clear the list of search results (if there were any from previous searches)